test/unit/test_regression.py

```diff
@@ -0,0 +1,102 @@
+import json
```

Comment:
This is great. What do you think about adding a small section in the readme with some info about the tests, how to run them, etc

Reply:
I was thinking about a dedicated document about development aspect (how to build and publish a package -- in the future, how to test, etc.) This way we'd had a readme that is all about usage of the software, and other that is focused on development/testing. I'll start one today and share.

Reply:
take a look at https://github.com/karol-brejna-i/pgn-tactics-generator/blob/chess-python-upgrade/DEVELOPMENT.md

Reply:
Excellent I like it.